### ISSUE TITLE
feat: Better support for Japanese/ Chinese keyboard

### ISSUE THREAD

**Issue by hiro-v** [janhq/jan#2585]
**Problem**
- Context: https://discord.com/channels/1107178041848909847/1223957838678724650/1223957838678724650
- Jan app currently does not support nicely for JP/ CN language while LLM is all about language
- This has negative effect to Jan goal for DAU

**Success Criteria**
- Allow users with those languages to seamlessly use Jan app
- How to reproduce:
   - Mac -> Language -> Add JP keyboard
   - Enable JP hiragana keyboard
   - Type `watashi` -> It will show `わたし` as suggestion. Normally user have to choose by using number then enter, but Enter here in Jan refers to submit message which is not intended

**Additional context**
- Check this for reference of implementation: https://dninomiya.github.io/form-guide/stop-enter-submit

**Comment by zhhanging** [comment]
In the `inputEvent` and `keyboardEvent`, there is an `isComposing` property, which can be used to determine whether the user is in input method's composing stage when triggering the input / keyboard event. https://developer.mozilla.org/en-US/docs/Web/API/InputEvent/isComposing. I think this will be an easy fix using this property. I could make a pr, but this issue is already assigned to imtuyethan. She will probably fix this soon.

**Comment by Van-QA** [comment]
Hi @zhhanging, please help us with the PR if possible 👍 
We value the help from contributors a lot.
Any ticket that you want, you can pick it up, you only need to change the assignee, and put a comment. 🙏

**Comment by Van-QA** [comment]
Many thank to the help of @zhhanging, the issue is now resolved as of Jan v0.4.10-368 ✅

**Comment by Van-QA** [comment]
hi @Enchante503, can you elaborate more on your case?

**Comment by imtuyethan** [comment]
@hiento09 Please guide @Enchante503 

> [＃3446](https://github.com/janhq/jan/issues/3446#issue-2483920660) Hi, The jan I installed should be a Dev repository, but the version is v0.1.0. I installed it using Git Clone. The method is as follows: Contributing>Instructions
> 
> I simply don't want to install it using Docker or any mysterious .deb format. If possible, please let me know how to install it using a Git clone method so that I can use versions like v0.5.2. Help me！

**Comment by hiento09** [comment]
Hi @Enchante503, the version 0.1.0 in the package.json will be updated when we run CI with the appropriate tag. If you want to build version v0.5.2 from source, please clone our source code and checkout to the tag v0.5.2. Thank you! cc @imtuyethan
